packages/cloudflare/README.md

```diff
@@ -161,6 +161,15 @@ The following example automatically generates `_routes.json` while including and
   });
 ```
 
+### `imageService`
+
+`imageService: 'external' | 'passthrough'`
```

Comment:
```suggestion
`imageService: "external" | "passthrough"`
default `"passthrough"`
```
Just keeping it consistent!

Reply:
The default value is not `passthrough`, it is a value the user shouldn't set, but the behaviour is in line with `passthrough`, so I'm unsure if we should put a default here.

Reply:
So the default is whichever service the main config has set or defaulted to?

Reply:
The default is whatever the user has as configuration, and if it is unsupported (sharp/squoosh) it is same as `passthrough`

Reply:
Ok, that makes sense. And this is more like an override of the user config.

Could the cloudflare image service be offered directly?
```ts
export default defineConfig({
  image: {
    service: '@astrojs/cloudflare/image-service',
  }
})
```
I'm guessing this would work already too.

Reply:
Yeah that code-snippet should work

Reply:
I'm wondering if it's a good idea to offer two ways of doing the same thing, but I don't feel strongly either way. I'll leave that upto you.

Reply:
I'm coming from: https://docs.astro.build/en/guides/integrations-guide/vercel/#imageservice
To make it "similar" ish, and I think you can do it both ways for Vercel too. But not 100% sure.

Reply:
Good point, this is similar to vercel. The documentation link reminds me, does it work same way as vercel in dev mode - with a configurable local alternative?

Reply:
So we currently use just "sharp" for local dev, since squoosh is deprecated right?
But I don't see any big issue with adding the devImageService config flag.

Reply:
I don't think the effort is necessary. LGTM.